client/src/components/DisplayTable.scss

```diff
@@ -1,5 +1,9 @@
 @import '../common_css/common-variables';
 
+table.display-table {
+  width: 99%;
+}
```

Comment:
What's this a fix for? Maybe add a "why" comment for future reference, because it looks like something I'd be tempted to delete.

Reply:
The border is outside the width which is set in bootstrap's `.table` to be 100%, which was putting a very slight scroll on the table. There's probably a better fix for this though? If not I can comment.

Reply:
Oh, gross, hate scrolls like that. Not in a CSS headspace right now so can't give a better fix off the top of my head.